File: hello world/Java/hello_world.java
package com.huawei.demo;
import com.huawei.services.runtime.Context;

public class hello_world {
    public static void main(String args[]) {}
    
    public String handler(Object event, Context context){
        //Get function name
        String f_name = "Hello world! Function name is " + context.getFunctionName();
        //Print function name
        System.out.println(f_name);

        return f_name;
    }
}

File: hello world/go/hello_world.go
package main

import (
    "fmt"
    "huaweicloud.com/go-runtime/go-api/context"
    "huaweicloud.com/go-runtime/pkg/runtime"
)

// Handler function
func Handler(payload []byte, ctx context.RuntimeContext) (interface{}, error) {
    //f_name := ctx.GetFunctionName()
	var f_name string = "Hello world! Function (Go1.x) name is " + ctx.GetFunctionName()
	fmt.Println(f_name)

    return f_name, nil
}

func main() { 
    runtime.Register(Handler) 
}

